CMakeLists.txt

```diff
@@ -12,3 +12,5 @@ qt_standard_project_setup()
 
 add_subdirectory(src/app)
 add_subdirectory(src/tests)
+add_subdirectory(src/app/receptionist)
+add_subdirectory(src/app/receptionist/test)
```

Comment:
I think it might be better to put it in CMakeLists.txt somewhere in tests folder.
It is not better to built parts of tests executables in "production" branch.
But still, it requires experiments an choosing best structure. To be discussed further.